src/olympia/stats/models.py

```diff
@@ -105,18 +105,6 @@ class Meta:
         db_table = 'theme_update_counts_bulk'
 
 
-class GlobalStat(models.Model):
-    id = PositiveAutoField(primary_key=True)
-    name = models.CharField(max_length=255)
-    count = models.IntegerField()
-    date = models.DateField()
-
-    class Meta:
-        db_table = 'global_stats'
```

Comment:
Added https://github.com/mozilla/addons-server/issues/11107 to drop that table.

Reply:
🙊 I keep forgetting to file those follow up issues